app/src/main/res/layout/fragment_continue_onboarding.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/contentView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/lime">
+
+    <LinearLayout
```

Comment:
fyi LinearLayout can be replaced with chains

Reply:
is it works faster? as far as I remember complex chains worked not so good if they were more then one

Reply:
I think it's just better to avoid layout nesting